src/main/resources/com/activestate/cloudfoundryjenkins/CloudFoundryPushPublisher/config.jelly

```diff
@@ -17,10 +17,11 @@
   </f:entry>
   <f:validateButton title="Test Connection" progress="Testing..." method="testConnection"
                     with="target,username,password,credentialsId,organization,cloudSpace,selfSigned"/>
-  <f:entry title="Reset app if already exists" field="resetIfExists">
-    <f:checkbox/>
+  <f:entry title="Cutover method">
```

Comment:
Add a `field` attribute here so that the data-bound constructor can pick up the value.